src/onegov/town6/locale/de_CH/LC_MESSAGES/onegov.town6.po

```diff
@@ -328,6 +331,9 @@ msgstr "Art:"
 msgid "The chat request got accepted."
 msgstr "Die Chat-Anfrage wurde angenommen."
 
+msgid "<dynamic element> <dynamic element>"
```

Comment:
This indicates that you either accidentally put a `i18:translate` somewhere or forgot `i18:name` for a replacement variable.